### PR TITLE
Adding the Re3gistry files

### PR DESCRIPTION
**Pull request by df-git**

### CODE REVIEW

**Comment by cvvergara** [comment]
@df-git 
https://travis-ci.com/github/OSGeo/OSGeoLive-doc/builds/186480701#L853
```
/home/travis/build/OSGeo/OSGeoLive-doc/build/doc/quickstart/re3gistry_quickstart.rst:69: WARNING: Title underline too short.

Introduction to the Re3gistry 2

================

/home/travis/build/OSGeo/OSGeoLive-doc/build/doc/quickstart/re3gistry_quickstart.rst:69: WARNING: Title underline too short.

Introduction to the Re3gistry 2

================

/home/travis/build/OSGeo/OSGeoLive-doc/build/doc/quickstart/re3gistry_quickstart.rst:78: WARNING: Title underline too short.

Users, roles and groups

==========

/home/travis/build/OSGeo/OSGeoLive-doc/build/doc/quickstart/re3gistry_quickstart.rst:78: WARNING: Title underline too short.

Users, roles and groups

==========

/home/travis/build/OSGeo/OSGeoLive-doc/build/doc/quickstart/re3gistry_quickstart.rst:93: WARNING: Explicit markup ends without a blank line; unexpected unindent.

```

The unexpected unindent problem is on the `.. tip::` command

How it should be:

```
.. tip::
   The tip goes indented

This is not part of the tip
```

How it is now
```
.. tip::
   The tip goes indented
This is part of the tip but its not indented

This is not part of the tip
```
I made locally the indentation and it looks like this:
image
I can read that you have  (i)

I played around locally: 
image

to get the (i) a bit more clear
image

File: doc/quickstart/re3gistry_quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Once the installation is finished, the user is redirected to the login page.
+
+**Migration option**
+
+In case the "Migration" option is chosen, the details of the database from the existing Re3gistry installation are requested.
+Fill the requested fields and click the "Save and continue" button. You will then get a page containing a summary of the data retrieved from the existing instance of the Re3gistry.
+Click on "Install" to begin the migration. An information page will be shown. Now you can close the browser and the system will notify you via e-mail using the address specified in the "Registry manager" definition step.
+
+Introduction to the Re3gistry 2
+================
```
Comment (cvvergara) [inline doc/quickstart/re3gistry_quickstart.rst:66]: Make 80 chars long preferably but at least make it at least as long as the title

File: doc/quickstart/re3gistry_quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+The Re3gistry editing interface allows to graphically manage the complete life-cycle of reference codes, such as code lists, enumerations, controlled vocabularies, taxonomies, thesauri or, simply, 'lists of things'.
+If you choose the "installation mode" from the wizard, the first time you log in to the system, you will have an empty registry. You can then start populating it using the tools provided by the editing interface.
+If you choose the "migration mode" from the wizard, you will find the registers imported from the older version of the tool.
+
+For detailed documentation on the editing interface, please read the `User manual: <https://github.com/ec-jrc/re3gistry/blob/master/documentation/user-manual.md>`
+
+Users, roles and groups
+==========
```
Comment (cvvergara) [inline doc/quickstart/re3gistry_quickstart.rst:75]: Make 80 chars long preferably but at least make it at least as long as the title

File: doc/quickstart/re3gistry_quickstart.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+The possible user roles that the Re3gistry 2 manages are:
+
+#. Registry manager
+#. Register owner
+#. Register manager
+#. Control body
+#. Submitting organisations
+
+.. tip::
```
Comment (cvvergara) [inline doc/quickstart/re3gistry_quickstart.rst:88]: Fix the tip, some hints are in a comment

**Review by cvvergara — CHANGES_REQUESTED** [review]

**Comment by cvvergara** [comment]
For translations this is needed:
Add the end of this file put the required lines for re3gistry, follow the example of mapcache 
https://github.com/OSGeo/OSGeoLive-doc/blob/master/.tx/config#L759

**Comment by kalxas** [comment]
Installer merged now

**Pull request merged**